apps/explorer/src/utils/getName.ts

```diff
@@ -1,16 +1,20 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { Base64DataBuffer } from '@mysten/sui.js';
+
 // TODO : Import from  SDK
 import { VALDIATOR_NAME } from '~/pages/validator/ValidatorDataTypes';
 
+const textDecoder = new TextDecoder();
+
 export function getName(rawName: string | number[]) {
     let name: string;
 
     if (Array.isArray(rawName)) {
         name = String.fromCharCode(...rawName);
     } else {
-        name = decodeURIComponent(atob(rawName));
+        name = textDecoder.decode(new Base64DataBuffer(rawName).getData());
```

Comment:
Didn't you just make this change? Curious why you're moving it back.

Reply:
Getting this issue `DOMException: Failed to execute 'atob' on 'Window'` on the latest local build